classes/Category.php

```diff
@@ -1720,6 +1720,10 @@ public function checkAccess($idCustomer)
      */
     public function updateGroup($list)
     {
+        // don't update group access if list is not set
+        if (!isset($list)) {
```

Comment:
```suggestion
        if ($list === null) {
```
But I' m not sure it's the correct solution, because if the array is empty the list is defined by 
`Configuration::get('PS_UNIDENTIFIED_GROUP'), Configuration::get('PS_GUEST_GROUP'), Configuration::get('PS_CUSTOMER_GROUP')`

Reply:
if the array is empty that means that the category is created or updated elsewhere of the backoffice gui because the group is mandatory : webservice, csv import, module, override, ...
At update time, the group don't have to be reset to the 3 default if the code don't care of it.
At creation time this weird stuff should be usefull : I'll make a new commit.

Reply:
Please replace the isset statement